### DISCUSSION TITLE
how to embed langfuse dashboard in other tool

### DISCUSSION THREAD

**Discussion by deepakdeore2004** [langfuse/langfuse#2574] [Support] [answered]
there is a custom in-house UI tool that has all the dashboards we need, is there a way to embed langfuse dashboard in another tool?

**Comment by marcklingen** [comment, marked as answer]
Embedding the dashboard is not possible, you could however fetch data from the langfuse apis and render it in a dashboarding tool. this might be a good starting point for simple timeseries charts: https://langfuse.com/docs/analytics/daily-metrics-api

**Reply by deepakdeore2004** [reply]
thanks for the reply @marcklingen, will explore on this

any plan to have a feature to make dashboard embedding easier with other tools

**Reply by marcklingen** [reply]
you can use the posthog integration (beta on langfuse cloud) if that's helpful for you, posthog dashboards can be easily embedded: https://langfuse.com/docs/analytics/posthog
